roachpb/data.go

```diff
@@ -661,12 +661,11 @@ func MakePriority(userPriority float64) int32 {
 	//
 	// For every random trial, if the normal priority is chosen >= max*x,
 	// the user priority loses 100% of the time; otherwise, wins 50% of
-	// the time. Therefore, x fraction of the time, loss is user(100%)
```

Comment:
btw, line 656 could be written as ` = (pri-1)/(pri+1)`, which is easy to recognize again below.